ansible/generate_terraform_register.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Generate register manifest for terraform
+  hosts: localhost
+  connection: local
+  vars:
+    workdir: '../aws/registers'
+    template: 'roles/terraform_config/templates/register_template.tf.j2'
+
+  vars_prompt:
+    - name: 'register'
+      prompt: 'Name of register'
+      private: no
+
+  tasks:
+    - name: generate template
```

Comment:
does this file need to be documented in the README? perhaps section around "Preparing new environment"?